cluster/cluster_fail.go

```diff
@@ -178,12 +171,12 @@ func (cluster *Cluster) MasterFailover(fail bool) bool {
 		dbhelper.MasterWaitGTID(cluster.master.Conn, cluster.oldMaster.GTIDBinlogPos.Sprint(), 30)
 	} else {*/
 	// Failover
-	cluster.LogPrintf(LvlInfo, "Waiting for candidate master to apply relay log")
+	cluster.LogPrintf(LvlInfo, "Waiting for candidate master %s to apply relay log", cluster.master.URL)
 	err = cluster.master.ReadAllRelayLogs()
 	if err != nil {
-		cluster.LogPrintf(LvlErr, "Error while reading relay logs on candidate: %s", err)
+		cluster.LogPrintf(LvlErr, "Error while reading relay logs on candidate %s: %s", cluster.master, err)
```

Comment:
Shouldn't this be `cluster.master.Name` or similar? currently it'll print the `ServerMonitor{}` to a string directly.

Reply:
Good catch